src/Apps/Settings/Routes/Purchases/Components/SettingsPurchasesRow.tsx

```diff
@@ -195,13 +220,7 @@ const SettingsPurchasesRow: FC<SettingsPurchasesRowProps> = ({ order }) => {
           <Text variant="sm-display">Order No.</Text>
 
           <Text variant="sm-display" color="black60">
-            {isOrderActive ? (
-              <RouterLink inline to={`/orders/${order.internalID}/status`}>
-                {order.code}
-              </RouterLink>
-            ) : (
-              order.code
-            )}
+            {getOrderLink(order)}
```

Comment:
Rather than a mix of function calls and object indexing to get color, label, text and link, I think I would get these values in the body of the FC - maybe using one single function at the top of the component body to resolve all of these values, or maybe just doing it inline.